addons/web/static/src/views/fields/many2many_tags_avatar/many2many_tags_avatar_field.xml

```diff
@@ -28,8 +28,13 @@
     <t t-name="web.KanbanMany2ManyTagsAvatarFieldTagsList" t-inherit="web.TagsList" t-inherit-mode="primary" owl="1">
         <xpath expr="//t[@t-foreach='visibleTags']" position="before">
             <t t-if="canDisplayQuickAssignAvatar">
-                <a t-on-click.stop.prevent="openPopover" tabIndex="-1" href="#" title="Assign"
-                   aria-label="Assign" class="fa fa-user-plus o_quick_assign o_m2m_avatar d-flex align-items-center"/>
+                <a t-on-click.stop.prevent="openPopover"
+                   tabIndex="-1"
+                   href="#"
+                   title="Assign"
+                   aria-label="Assign"
```

Comment:
According to https://github.com/odoo/odoo/wiki/SCSS-coding-guidelines, can you set `btn-link` before `d-flex`?